app/models/sample.rb

```diff
@@ -75,6 +75,7 @@ def pipeline_command
     script_name = File.basename(IdSeqPipeline::S3_SCRIPT_LOC)
     batch_command = "aws s3 cp #{IdSeqPipeline::S3_SCRIPT_LOC} .; chmod 755 #{script_name}; " \
       "INPUT_BUCKET=#{sample_input_s3_path} OUTPUT_BUCKET=#{sample_output_s3_path} " \
+      "STAR_GENOME=#{s3_star_index_path} BOWTIE2_GENOME=#{s3_bowtie2_index_path} " \
```

Comment:
this can be a problem if user doesn't specify it. I think you will only add this in when the index paths are set

Reply:
Isn't that problem dealt with in aws_batch/pipeline.py? There's a default in the pipeline. So far we've never set some of the other ENV variables (e.g. SAMPLE_SEQUENCER) and it always worked, so probably won't cause failure?